modules/aruco/src/dictionary.cpp

```diff
@@ -377,8 +378,8 @@ static int _getSelfDistance(const Mat &marker) {
 /**
  */
 Ptr<Dictionary> generateCustomDictionary(int nMarkers, int markerSize,
-                                         const Ptr<Dictionary> &baseDictionary) {
-
+                                         const Ptr<Dictionary> &baseDictionary, int randomSeed) {
+    setRNGSeed(randomSeed); // #1586
```

Comment:
In internal implementation we can pass `cv::RNG& rng` instead of initial seed value.

Perhaps we should not touch OpenCV seed in this case (this code is data generator, not a stochastic algorithm, like RANSAC).

Reply:
good idea, that would also solve the setRNGSeed() problem.
plan now: use size_t and cv::RNG, and pass on the latter internally.